### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by DmitryPR**
This PR is mostly about adding more information for the integrator by introducing FAQ and Videos

### CODE REVIEW

File: docs/Videos.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Videos
+
+* Find out how to [quickstart][0] using cocoapods and carthage.
```
Comment (ostap0207) [inline docs/Videos.md]: These lines seem to be referring to a different person and not consistent.
Perhaps something like:
```
Find out how to:
* [quickstart][0] using cocoapods and carthage
* [setup][1] the SDK to enable push notification and rich media
* [integrate][2] the SDK, with real code examples
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Frequently Asked Questions
+
+This is a list of questions we frequently get and problems that are often encountered. 
+
+* [Example app not compiling](#example-app-not-compiling)
+* [Why do we include a custom socket.io library](#why-do-we-include-a-custom-socket-library)
+
+## Example app not compiling
+
+There is an Example app where the sdk is integrate using cocoapods. Afther checking out
```
Comment (ostap0207) [inline docs/FAQ.md]: `SDK`

Comment (ostap0207) [reply docs/FAQ.md]: `integrate` -> `integrated`

Comment (ostap0207) [reply docs/FAQ.md]: `Afther` -> `After`

File: docs/Video.md [hunk 3]
```diff
@@ -14,6 +14,12 @@ Answer [setup][1] questions, enabling push, rich media and configuration options
 
 How to [integrate][2] the client library with real code examples.
 
+
+## Screenshare
+
+How to start [screenshare][3] and see visitor's screen.
```
Comment (ostap0207) [inline docs/Video.md]: How to start screen sharing?

**Comment by DmitryPR** [comment]
Updated the section

**Comment by justindipietro** [comment]
The screenshare video needs to be fixed:
1. It shows the CoBrowsing area not loading, and then even failing with an error message
2. When the visitor types back to the operator it shows in the screen sharing but not the chat area

**Comment by DmitryPR** [comment]
I cannot do anything regarding co browsing and the "Hello" message is actually received by the Operator in the video

**Comment by DmitryPR** [comment]
!squash

**Comment by DmitryPR** [comment]
!merge

**Pull request merged**